calculator.js

```diff
@@ -1,10 +1,91 @@
+/*jshint esversion: 6 */
 /**
  * Declare a function named `calculatorModule`
  * this function will have two private variables declared inside of it.
  * @variable PRIVATE { Number } `memory`
  * @variable PRIVATE { Number } `total`
  * @return {object} `calculator` object that can be used
  */
+function calculatorModule(){
+  let memory = 0;
+  let total = 0;
+
+  function load(x){
+  if(typeof x !== 'number'){
+    throw new Error("Invalid");
+  } else {
+    total = x;
+    }
+  return total;
+    }
+  function getTotal(){
+    return total;
+  }
+  function add(x){
+    if(typeof x !== 'number'){
+    throw new Error("Invalid");
+    } else {
+    total = x + total;
+    }
+    return total;
+  }
+  function subtract(x){
+    if(typeof x !== 'number'){
+    throw new Error("Invalid");
+    } else {
+    total = total - x;
+    }
+    return total;
+  }
+  function multiply(x){
+    if(typeof x !== 'number'){
+    throw new Error("Invalid");
+    } else {
+    total = total * x;
+    }
+    return total;
+  }
+  function divide(x){
+    if(typeof x !== 'number'){
+    throw new Error("Invalid");
+    } else {
+    total = total / x;
+    }
+    return total;
+  }
+  function recallMemory(){
+    return  memory;
+  }
+  function saveMemory(){
+    memory = total;
+    return memory;
+  }
+  function clearMemory(){
+    memory = total - memory;
+    return memory;
+  }
+    return {
+    load : load,
+    getTotal : getTotal,
+    add : add,
+    subtract : subtract,
+    multiply : multiply,
+    divide : divide,
+    recallMemory : recallMemory,
+    saveMemory : saveMemory,
+    clearMemory : clearMemory,
+  };
+
+}
```

Comment:
After you have your code complete and have your tests passing. You should remove your console logs.